api-module-library/slack/manager.js

```diff
@@ -150,7 +195,7 @@ class Manager extends ModuleManager {
 
     async updateOrCreateCredential() {
         const workspaceInfo = await this.api.authTest();
-        const externalId = get(workspaceInfo, 'team_id');
+        const externalId = workspaceInfo['bot_user_id'] ? get(workspaceInfo, 'team_id') : get(workspaceInfo, 'user_id');
```

Comment:
I'm guessing that this is determining whether we are updating a team or user credential? Might be worth a comment to indicate

Reply:
Changed the code to indicate, instead of adding a comment